src/Build.UnitTests/Evaluation/Evaluator_Tests.cs

```diff
@@ -4675,11 +4670,6 @@ public void VerifyPropertyTrackingLoggingNone()
                         .EnvironmentVariableName
                         .ShouldBe("DEFINED_ENVIRONMENT_VARIABLE2");
 
-                    logger
-                        .AllBuildEvents
-                        .OfType<PropertyReassignmentEventArgs>()
-                        .ShouldBeEmpty();
```

Comment:
Instead of removing this could you validate that it has the correct single value, like the env var block above?